src/objects.rs

```diff
@@ -18,6 +18,45 @@ use collection::{Id, Idx};
 use utils::*;
 use chrono;
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum ObjectType {
+    #[serde(rename = "stop_area")]
```

Comment:
You can do a `#[serde(rename_all = "snake_case")]` to do almost everything (except trip)

Reply:
```rust
#[derive(Serialize, Deserialize, Debug, Clone)]
#[serde(rename_all = "snake_case")]
pub enum ObjectType {
    StopArea,
    StopPoint,
    Network,
    Line,
    Route,
    #[serde(rename = "trip")]
    VehicleJourney,
    StopTime,
    LineGroup,
}
```